src/pages/jaewonhwang/Main/Main.js

```diff
@@ -0,0 +1,340 @@
+                  <li>
+                    <img
+                      src="https://lh3.googleusercontent.com/ogw/ADGmqu-5wYZxIpZOgJh5ZS4YIj9YtBMhSA1hn7578X5A=s32-c-mo"
+                      alt="이미지"
+                    />
+                    <p>r_yuhyeju</p>
+                  </li>
+                  <li>
+                    <img
+                      src="https://lh3.googleusercontent.com/ogw/ADGmqu-5wYZxIpZOgJh5ZS4YIj9YtBMhSA1hn7578X5A=s32-c-mo"
+                      alt="이미지"
+                    />
+                    <p>kxngg.94</p>
+                  </li>
+                </ul>
+              </section>
+              <section className="post">
+                <div className="post_header">
+                  <img
+                    src="https://lh3.googleusercontent.com/ogw/ADGmqu-5wYZxIpZOgJh5ZS4YIj9YtBMhSA1hn7578X5A=s32-c-mo"
+                    alt="이미지"
+                  />
+                  <span>
+                    <p className="post_name">summer_01</p>
+                    <p className="city">NYC</p>
+                  </span>
+                  <i className="fas fa-ellipsis-h"></i>
+                </div>
+                <div className="post_img">
+                  <img
+                    src="https://images.unsplash.com/photo-1514477917009-389c76a86b68?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1322&q=80"
+                    alt="이미지"
+                  />
+                </div>
+                <div className="post_icon">
+                  <span>
+                    <i className="fas fa-heart"></i>
+                    <i className="far fa-comment"></i>
+                    <i className="fas fa-external-link-alt"></i>
+                  </span>
+                  <i className="far fa-bookmark"></i>
+                </div>
+                <div className="post_like">
+                  <img
+                    src="https://lh3.googleusercontent.com/ogw/ADGmqu-5wYZxIpZOgJh5ZS4YIj9YtBMhSA1hn7578X5A=s32-c-mo"
+                    alt="이미지"
+                  />
+                  <p>님 외 8명이 좋아합니다.</p>
+                </div>
+                <div className="post_text">
+                  <h5>summer_01</h5>
+                  <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    Nisi est neque omnis illum quidem ipsum reiciendis, natus ut
+                    iusto, unde minus, in voluptas nemo minima necessitatibus?
+                    Cupiditate sed mollitia nemo.
+                  </p>
+                </div>
+                <div className="post_review">
+                  <ul>
+                    {this.state.commentList.map((commentText) => {
+                      return (
+                        <Content
+                          userId={commentText.userId}
+                          content={commentText.content}
+                        />
+                      );
+                    })}
```

Comment:
```suggestion
                    {this.state.commentList.map(commentText => (
                        <Content
                          userId={commentText.userId}
                          content={commentText.content}
                        />
                      )
                    )}
```
arrow function에서 리턴만 할 경우에는 {} 가 아니라 바로 () 로 감싸서 불필요한 괄호를 생략할 수 있습니다.
마찬가지로 하나의 인자만 받을 경우에는 인자부분에도 ()를 생략할 수 있습니다.

Reply:
이 부분 배열을 키값으로 가지고 있는 state는 return을 꼭써줘야하나요? 리턴을 지우고 해봤더니 자꾸오류가 납니다!